docs/en/infraops/installation.asciidoc

```diff
@@ -67,6 +67,8 @@ To populate the *Hosts* view and add logs, enable:
 * {filebeat-ref}/filebeat-module-system.html[{filebeat} `system` module]
 * {filebeat-ref}/filebeat-modules.html[Other {filebeat} modules] needed for
 your environment, such as `apache2`, `redis`, and so on
+* {metricbeat-ref}/add-host-metadata.html[{metricbeat} `add_host_metadata` processor]
```

Comment:
Maybe you should indicate parenthetically that these are enabled by default? My concern is that users be frustrated if they spend time learning how to enable the processors but then find out that they are already enabled. 

so maybe do something like:

```
* {metricbeat-ref}/add-host-metadata.html[{metricbeat} `add_host_metadata` processor] (enabled by default)
* {metricbeat-ref}/add-cloud-metadata.html[{metricbeat} `add_cloud_metadata` processor] (enabled by default)
```

I think you could do the same for the metricbeat system module, if I'm not mistaken.

Reply:
Good point @dedemorton, I'll make the changes